md/bdm-in-rest-api.md

```diff
@@ -216,6 +216,94 @@ class CarManagement implements RestApiController {
     }
 ```
 
+### Returning the whole object with an API link load in the lazy fields
+The idea is to create custom Json serializer.  
+A custom Json serializer is a class which extends *com.fasterxml.jackson.databind.JsonSerializer*. There is a method *serialize* to implement, which has the responsability to serialize the input model into Json.  
+The custom Json serializer has to come with two other classes:  
+
+ - An object wrapper class, which extends *com.fasterxml.jackson.databind.*
+ - A module class, which extends *com.fasterxml.jackson.databind.module.SimpleModule*
+ 
+The module contains the custom serializer, and the wrapper contains the module. At the end, in your rest API endpoint, you interact with the wrapper.
+
+Here is an implementation example for an object Employee which has a lazy attribute of type Address:  
```

Comment:
I wouldn't introduce a new BO but use the Car example as in the rest of this how-to.
I understand that you need an attribute which is a multiple BO, maybe we have to enrich our example